tf-generator/aws-services/rds.go

```diff
@@ -213,6 +213,9 @@ func (r *Rds) Generate(config *common.Config, client *duplosdk.Client) (*common.
 				rdsBody.SetAttributeValue("multi_az",
 					cty.BoolVal(rds.MultiAZ))
 
+				rdsBody.SetAttributeValue("storage_type", cty.StringVal(rds.StorageType))
```

Comment:
Add if condition for empty in case of string, zero in case of number if these attributes are not required.

Reply:
Done